portfolio/src/main/webapp/index.html

```diff
@@ -47,7 +47,8 @@ <h2> Fun Fact Gallery </h2>
             <!-- image on Colombia -->
             <div class="column"> 
                 <div class="content"> 
-                    <img src="https://media.nomadicmatt.com/colombiaguide.jpg" alt="Colombia" >
+                    <img src="https://media.nomadicmatt.com/colombiaguide.jpg" 
+                    alt="Colombia" >
```

Comment:
Two last things here, alt should be indented from img, and you don't need an extra space at the end before the closing bracket.
For example:

```
<img src="..."
     alt="...">
```